<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PStatsDialog</class>
 <widget class="QDialog" name="PStatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>All Documents:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="labelTotalParagraph">
                 <property name="text">
                  <string>Paragraph Count:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelTotalWC">
                 <property name="text">
                  <string>Word Count:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelTotalCharWS">
                 <property name="text">
                  <string>Character Count (Whitespaces):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelTotalChar">
                 <property name="text">
                  <string>Character Count (No Whitespace):</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="labelIntTotalParagraph">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIntTotalWC">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIntTotalCharWS">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIntTotalChar">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
              <zorder>widget_4</zorder>
              <zorder>labelIntTotalParagraph</zorder>
              <zorder>labelIntTotalCharWS</zorder>
              <zorder>labelIntTotalChar</zorder>
              <zorder>labelIntTotalWC</zorder>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Current Document:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="labelCurrentParagraph">
                 <property name="text">
                  <string>Paragraph Count:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCurrentWC">
                 <property name="text">
                  <string>Word Count:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCurrentCharWS">
                 <property name="text">
                  <string>Character Count (Whitespaces):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelCurrentChar">
                 <property name="text">
                  <string>Character Count (No Whitespace):</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="labelIntCurrentParagraph">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIntCurrentWC">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIntCurrentCharWS">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelIntCurrentChar">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
